### PR TITLE
Add `docker_migrate` to README

### PR DESCRIPTION
**Pull request by nemaniarjun**
##### Description
When facing module not found errors, running this command fixes them, needs to be run only once for
new containers.

##### Checklist
- [x] Linter status: 100% pass
- [x] Changes don't break existing behavior
- [x] Tests coverage hasn't decreased

##### Affected core subsystem(s)
Docs

##### Refers/Fixes
Fixes: #283

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,21 @@ At this point we should be able to run the dApp locally
...
 ```
 
+### NOTE:
+
+If you face the following error:
+```
+./src/Contracts.js
+Module not found: Can't resolve '../build/contracts/InitialAllocationCollateralToken' in '/app/src'
+```
+
```
Comment (perfectmak) [inline README.md]: @nitinrgupta You might want to add an extra side comment that this usually happens when starting the app with docker using docker-compose.

Or better still maybe we should just add an `Option 3 - local test rpc (Docker)` with explicit instruction on how to start the dapp with docker.

Comment (nitinrgupta) [reply README.md]: @perfectmak I think you meant to tag @nemaniarjun :)

Comment (perfectmak) [reply README.md]: Oops! @nemaniarjun.

Comment (nemaniarjun) [reply README.md]: @perfectmak That seems like a fair idea! I will update this ASAP!

**Comment by nemaniarjun** [comment]
@perfectmak review please :grin:

**Pull request merged**